src/Common/src/System/Collections/Generic/LowLevelQueue.cs

```diff
@@ -0,0 +1,288 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+/*=============================================================================
+**
+** Private version of Queue<T> for internal System.Private.CoreLib use. This
+** permits sharing more source between BCL and System.Private.CoreLib (as well as the
+** fact that Queue<T> is just a useful class in general.)
+**
+** This does not strive to implement the full api surface area
+** (but any portion it does implement should match the real Queue<T>'s
+** behavior.)
+**
+** This file is a subset of System.Collections\System\Collections\Generics\Queue.cs
+** in CoreFX and should be kept in sync with that file.
+**
+=============================================================================*/
+
+using System.Diagnostics;
+
+namespace System.Collections.Generic
+{
+    // A simple Queue of generic objects.  Internally it is implemented as a 
+    // circular buffer, so Enqueue can be O(n).  Dequeue is O(1).
```

Comment:
Yea, just copy/pasted from CoreFX. I'm going to remove this file as it's no longer necessary.